### PR TITLE
New Banner

### PR DESCRIPTION
**Pull request by elisacarrillo**
Hi Kflan!
Thought you might enjoy this :) 
image

### CODE REVIEW

**Comment by elisacarrillo** [comment]
what it currently looks like: 
image

**Comment by elisacarrillo** [comment]
image
30%

File: index.md
Comment (elisacarrillo) [inline index.md:1]: changed to associate professor from assistant professor

**Comment by wadefagen** [comment]
@elisacarrillo -- Why so small?

Just a mock-up in paint, but could you make it fill the entire vertical space?
image

**Comment by elisacarrillo** [comment]
image

**Pull request merged**